### ISSUE TITLE
Repeating days

### ISSUE THREAD

**Issue by alextakitani** [themouette/jquery-week-calendar#156]
On the week of 20 - oct it doesn't shows

21, 22, 23, 24, 25, 26 and 27.

tested on chrome and firefox, last stable versions.

**Comment by edbizarro** [comment]
yeah, I have a similar problem with the same day, my calendar shows me saturday'20 and duplicate that day, now I have saturday appearing 2 times as sunday'20
